src/deploy/NVA_build/noobaa_init.sh

```diff
@@ -116,8 +116,6 @@ prepare_mongo_pv() {
   ${KUBE_PV_CHOWN} mongo
 
   mkdir -p ${dir}
```

Comment:
change dir above to `/data/mongo/cluster/shard1`
also, look for all other places that use `mongo_data`